migrations/1_initial_migration.js

```diff
@@ -87,6 +114,40 @@ async function deployDssDeploy(deployer) {
 	)
 }
 
+// Deploy the Ether-direct collateral (no wrapped ETH)
+async function deployCollateral(deployer, dssDeploy, vatContract) {
+	const ethJoin = await deployer.deploy(artifacts.require('ETHJoin'), vatContract.address, ETH_COLLATERAL_ID)
+	await dssDeploy.deployCollateral(ETH_COLLATERAL_ID, ethJoin.address, FAKE_PRICE_FEED_ADDRESS)
+	await vatContract.file(ETH_COLLATERAL_ID, web3.utils.asciiToHex("line"), DEBT_CEILING)
+	await vatContract.file(ETH_COLLATERAL_ID, web3.utils.asciiToHex("spot"), FIXED_FEED_PRICE)
+	await vatContract.file(web3.utils.asciiToHex("Line"), DEBT_CEILING)
+	return ethJoin
+}
+
+async function createVaultAndWithdrawDai(account, vatContract, ethJoin, daiJoinAddress, daiAddress) {
```

Comment:
This is getting into the realm of "maybe this should be part of the test suite, not the deploy suite".  The line is already fuzzy, and it doesn't really matter too much.

Reply:
I agree, i wrote on the main description "Seems like it could go into a few Truffle phases, in a perfect world."